### PR TITLE
SUR-336 - Added uat changes

### PR DESCRIPTION
**Pull request by BabuTechBsf**
### Description

Added Uat fixes from : https://brainstormforce.atlassian.net/browse/SUR-331

### Checklist:

-   [ ] My code is tested
-   [x] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [ ] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/progress-steps/progress-steps.tsx [hunk 1]
```diff
@@ -49,10 +49,10 @@ export interface ProgressStepsProps extends ProgressCommonProps {
 	size?: 'sm' | 'md' | 'lg';
 	/** Defines the type of layout. */
 	type?: 'inline' | 'stack';
-	/** Defines the current step number. */
+	/** Defines the current step number. '-1' keeps all steps completed. */
```
Comment (jaieds) [inline src/components/progress-steps/progress-steps.tsx:52]:
```suggestion
	/** Defines the current step number. `-1` keeps all steps completed. */
```

File: src/components/pagination/pagination.stories.tsx [hunk 2]
```diff
@@ -51,8 +51,47 @@ Basic.args = {
 export const Small = Template.bind( {} );
```
Comment (jaieds) [inline src/components/pagination/pagination.stories.tsx:51]: Pagination `xs, lg` sizes are missing. Need to fix it from the component pop type.

File: src/components/menu-item/menu.stories.tsx [hunk 3]
```diff
@@ -99,7 +98,7 @@ const Template: StoryFn = ( args ) => (
 	</Menu>
 );
 
-export const CombinedMenu = Template.bind( {} );
+export const CombinedMenu: Story = Template.bind( {} );
 
 CombinedMenu.args = {
 	size: 'md',
```
Comment (jaieds) [inline src/components/menu-item/menu.stories.tsx:104]: Show arrow on hover not working? When `showArrowOnHover = false` it does not show the icon.

File: src/components/pagination/pagination.stories.tsx (hunk 2 above)
Comment (BabuTechBsf) [reply src/components/pagination/pagination.stories.tsx:51]: added

File: src/components/menu-item/menu.stories.tsx (hunk 3 above)
Comment (BabuTechBsf) [reply src/components/menu-item/menu.stories.tsx:104]: this setting only for showing on hover. It stops on hover based on the boolean.

**Review by jaieds — APPROVED** [review]

**Pull request merged**